src/packages/cli/package.json

```diff
@@ -141,7 +144,7 @@
     "format": "prettier --write .",
     "lint": "eslint --fix --ext .js,.ts .",
     "download": "rm -f download-done && node scripts/updateTag.js && node scripts/download.js || echo \"\"",
-    "tsc": "tsc -d && bash scripts/copy-runtime-dist.sh",
+    "tsc": "tsc -d -p tsconfig.build.json && bash scripts/copy-runtime-dist.sh",
```

Comment:
Jest was failing with static errors (supplied by `ts-jest`) that I wasn't seeing in my editor. The solution was to have a general `tsconfig` for all TS code in the project, and then another for just the build step.